lib/Panda/Builder.pm

```diff
@@ -104,20 +107,30 @@ method build($where, :$bone) {
                 #    next;
                 #}
                 say "Compiling $file to {comptarget}";
-                my $cmd    = "$*EXECUTABLE --target={comptarget} "
-                           ~ "--output=$dest $file";
-                $output ~= "$cmd\n";
-                my $handle = pipe("$cmd 2>&1", :r);
-                for $handle.lines {
-                    .chars && .say;
-                    $output ~= "$_\n";
-                }
-                my $passed = $handle.close.status == 0;
+
+                my $proc = Proc::Async.new($*EXECUTABLE, "--target={comptarget}", "--output=$dest", $file);
+                $output ~= "$*EXECUTABLE --target={comptarget} --output=$dest $file\n";
+                $proc.stdout.tap(-> $chunk {
+                    print $chunk;
+                    $output ~= $chunk;
+                    $stdout ~= $chunk;
+                });
+                $proc.stderr.tap(-> $chunk {
+                    print "\e[31;1m$chunk\e[0m";
```

Comment:
Whoops, that's some code I was using for debugging!  I'll submit a PR to remove it.